app/admin/golem/monitor/page.tsx

```
@@ -0,0 +1,224 @@
+
+  if (error && !dashboard) {
+    return (
+      <div className="text-center py-20 text-red-400">
+        <p>Failed to load monitor dashboard: {error}</p>
+      </div>
+    );
+  }
+
+  if (!dashboard) return null;
+
+  const serviceMap = useMemo(() => {
```

Biome (2.3.13)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)